templates/pakwheels_register.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+{% if messages %}
+<div class="span12">
+    {% for message in messages %}
+        <div class="alert alert-{{ message.tags }}">
+            {{ message|safe }}
+        </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+
+ <form method="post">
+        {% csrf_token %}
+            <table>
```

Comment:
dedent this section to same level as csrf tag since the csrf tag is not conditional tag, i.e. `if`, `else`, `for`

```django
<form method="post">
    {% csrf_token %}
    <table>
    </table>
</form>